packages/web/src/components/Home/index.tsx

```diff
@@ -16,33 +17,25 @@ export default function Home() {
             body: `Load a CSV, Parquet or JSON (data source)
             that includes your dataset location to get started.`,
             action: (
-                <DefaultButton
-                    className={classNames(styles.button)}
-                    styles={{
-                        label: styles.buttonLabel,
-                        icon: styles.buttonIcon,
-                    }}
-                    ariaLabel="Upload file"
-                    iconProps={{ iconName: "Upload" }}
-                    title="Upload file"
-                    text="UPLOAD FILE"
-                />
+                <Link to="app">
```

Comment:
It seems like with the new button styles we might need to add `text-decoration: none;` to the link part
![image](https://github.com/AllenInstitute/aics-fms-file-explorer-app/assets/24441706/0fe4f75f-cb7c-40a3-b5e3-35bc2821aad4)